SI/Sem.02/Pract.02/task_4_p02.cpp

```diff
@@ -0,0 +1,43 @@
+﻿
+#include <iostream>
+
+int main()
+{
+    int result = 0;
+    std::cout << "Result: ";
+    std::cin >> result;
+    if (result >= 0 && result <= 100)
+    {
+        if (result < 40)
+        {
+            std::cout << "Grade : 2";
+        }
+        if (result >= 40 && result < 60)
+        {
+            std::cout << "Grade : 3";
+        }
+        if (result >= 60 && result < 70)
+        {
+            std::cout << "Grade : 4";
+        }
+        if (result >= 70 && result < 80)
+        {
+            std::cout << "Grade : 5";
+        }
+        if (result >= 80 && result < 90)
+        {
+            std::cout << "Grade : 6";
+        }
+        if (result >= 90)
+        {
+            std::cout << "Grade : 6 + ";
```

Comment:
This should be an else if construction
With else if you will prevent making useless checks and
        if (result < 40) {
            std::cout << "Grade : 2";
        }
        if (result >= 40 && result < 60) {
            std::cout << "Grade : 3";
        }
and here you you won't have to write result >= 40 in the second if